src/utils/constants.ts

```diff
@@ -85,4 +85,17 @@ const StorageKeys = {
   TOAST: 'toast',
 };
 
-export {Constants, StorageKeys};
+/**
+ * Local storage keys for intervention impressions.
+ */
+const ImpressionStorageKeys = {
```

Comment:
nit: can we use consistent type names as https://github.com/subscriptions-project/swg-js/blob/827e4418685a75ab0402a3ca15cfc0a010840890/src/runtime/audience-action-flow.ts#L67?
REGISTRATION_WALL, NEWSLETTER_SIGNUP, REWARDED_SURVEY, REWARDED_AD

Reply:
Sounds good, is `REWARDED_AD` something we want to add now?

Reply:
Will add the enum now but not the event mapping as the Impression enum for rewarded ads has not yet been created.